setup-deps.sh

```diff
@@ -330,15 +325,14 @@ fi
 cd "$NOPCOMMERCE_DIR"
 cp -r README.md "$DOCS_DIR/Nopcommerce"
 for file in "$DOCS_DIR"/Nopcommerce/*.md; do
-  update_external "$file" https://github.com/btcpayserver/nopcommerce "$DOCS_DIR"/Nopcommerce/ main
+  update_external "$file" https://github.com/btcpayserver/nopcommerce https://github.com/btcpayserver/nopcommerce/edit/main "$DOCS_DIR"/Nopcommerce/
 done
 
 # Wix
 echo "Setup dependency: Wix"
 
-if [ -f "$DOCS_DIR/Wix.md" ]; then
-  rm "$DOCS_DIR/Wix.md"
-fi
+rm -rf "$DOCS_DIR/Wix"
+mkdir -p "$DOCS_DIR/Wix"
```

Comment:
Hey @dennisreimann  there is a duplicate command here

Reply:
@TChukwuleta @dennisreimann pushed commit to resolve this 9d5abd62713367c975b19b28dc22f43cd17fde4b

Reply:
Thanks for spotting and fixing!